gef-ui/templates/_macros/application-activity/activity-html.njk

```diff
@@ -0,0 +1,45 @@
+{% from "govuk/components/tag/macro.njk" import govukTag %}
+{% import './facility.njk' as facilityActivity %}
+{% import './deal-cancelled.njk' as dealCancelledActivity %}
+
+{% macro render(params) %}
+
+  {% set title = params.title %}
+  {% set dealId = params.dealId %}
+  {% set ukefDealId = params.ukefDealId %}
+  {% set previousStatus = params.previousStatus %}
+  {% set facilityType = params.facilityType %}
+  {% set facilityId = params.facilityId %}
+  {% set ukefFacilityId = params.ukefFacilityId %}
+  {% set maker = params.maker %}
+  {% set checker = params.checker %}
+
+  {% if title === 'Bank facility stage changed' %}
+
```

Comment:
Does this value come from the database? If so is this the value `title` takes on existing deals?

Reply:
It's saved as `label` in the DB as per other activities. The MOJ component has a `title` element so in the UI's controller we map `title` to `label`.

Reply:
Marking as resolved for merge and to help clear the backlog.

Happy to change this post-merge if required @MarRobSoftwire